.travis.yml

```diff
@@ -1,9 +1,22 @@
 # Rust FTW
 language: rust
+cache: cargo
 
 # We don't need sudo to build the project.
 sudo: false
 
+# Clippy matrix
+matrix:
+  fast_finish: true
+  include:
+    - rust: nightly-2018-01-12
```

Comment:
Por que incluir `nightly`?

Reply:
pra conseguir rodar o clippy, pq ele usa o toolchain nightly

Reply:
Viu que o run do Travis levou 9 minutos? Isso é normal?

Reply:
Pelo o que to vendo, grande parte da demora do build é por conta das imagens do MacOS